.github/workflows/static_analysis.yaml

```diff
@@ -0,0 +1,27 @@
+name: Snyk Static Analysis Scan
```

Comment:
These actions both execute on the same triggers. Any reason to not combine them?

Reply:
I have them in separate jobs because they are two different scans.  They may grow in complexity as our needs evolve so I just thought it was simpler this way.